cmd/main.go

```diff
@@ -279,7 +279,7 @@ func electMaster(mgr manager.Manager, nsxClient *nsx.Client) {
 
 func main() {
 	log.Info("Starting NSX Operator")
-	mgr, err := ctrl.NewManager(ctrl.GetConfigOrDie(), ctrl.Options{
+	mgr, err := ctrl.NewManager(pkgutil.GetConfig(), ctrl.Options{
```

Comment:
So the API server address switch only can occur in the startup stage, right? Then if the eth1 down during the NSX operator runtime, what will happen?

Reply:
I've checked this in HA mode, and found operator will restart after eht1 down automatically because the lease renewal failed.